### PR TITLE
Remove unused state_change table

### PR DESCRIPTION
**Pull request by hustodemon**
## What does this PR change?
 
 Remove an table used for old monitoring (that has already been removed in the past).
 
 ## GUI diff
 
 No difference.
 
 Before:
 
 After:
 
 - [x] **DONE**
 
 ## Documentation
 - No documentation needed: Removing unused table
 
 - [x] **DONE**
 
 ## Test coverage
 - No tests: We don't have tests for this code area
 
 - [x] **DONE**
 
 ## Links
 
 Tracks https://github.com/SUSE/spacewalk/issues/6213
 
 - [x] **DONE**
 
 ## Changelogs
 
 If you don't need a changelog check, please mark this checkbox:
 
 - [x] No changelog needed
 
 If you uncheck the checkbox after the PR is created, you will need to re-run `changelog_test` (see below)
 
 
 ## Re-run a test
 
 If you need to re-run a test, please mark the related checkbox, it will be unchecked automatically once it has re-run:
 
 - [ ] Re-run test "changelog_test"
 - [ ] Re-run test "backend_unittests_pgsql"
 - [ ] Re-run test "java_lint_checkstyle"		 
 - [ ] Re-run test "java_pgsql_tests"		 
 - [ ] Re-run test "ruby_rubocop"
 - [ ] Re-run test "schema_migration_test_oracle"
 - [ ] Re-run test "schema_migration_test_pgsql"		 
 - [ ] Re-run test "susemanager_unittests"
 - [ ] Re-run test "javascript_lint"

### CODE REVIEW

File: schema/spacewalk/upgrade/susemanager-schema-4.0.9-to-susemanager-schema-4.0.10/150-drop_state_change.sql.postgresql [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+-- FOR A PARTICULAR PURPOSE. You should have received a copy of GPLv2
+-- along with this software; if not, see
+-- http://www.gnu.org/licenses/old-licenses/gpl-2.0.txt.
+--
+-- Red Hat trademarks are not licensed under GPLv2. No permission is
+-- granted to use or replicate Red Hat trademarks that are incorporated
+-- in this software or its documentation.
+--
+
+DROP TABLE IF EXISTS state_change;
```
Comment (mcalmer) [inline schema/spacewalk/upgrade/susemanager-schema-4.0.9-to-susemanager-schema-4.0.10/150-drop_state_change.sql.postgresql]: I think you need to drop the index as well? Or is it dropped automatically when the table is gone?

Comment (hustodemon) [reply schema/spacewalk/upgrade/susemanager-schema-4.0.9-to-susemanager-schema-4.0.10/150-drop_state_change.sql.postgresql]: On postgres it works fine. But I'll check on oracle, just to be sure.

Comment (juliogonzalez) [reply schema/spacewalk/upgrade/susemanager-schema-4.0.9-to-susemanager-schema-4.0.10/150-drop_state_change.sql.postgresql]: Oracle? We don't have Oracle anymore :)

Comment (hustodemon) [reply schema/spacewalk/upgrade/susemanager-schema-4.0.9-to-susemanager-schema-4.0.10/150-drop_state_change.sql.postgresql]: Ok, oracle is also safe.

Comment (hustodemon) [reply schema/spacewalk/upgrade/susemanager-schema-4.0.9-to-susemanager-schema-4.0.10/150-drop_state_change.sql.postgresql]: Ah, I see - all the migration files are empty now (last time I added the scripts it wasn't the case :smile:  ). Even better.

Comment (mcalmer) [reply schema/spacewalk/upgrade/susemanager-schema-4.0.9-to-susemanager-schema-4.0.10/150-drop_state_change.sql.postgresql]: And the index is really gone on postgresql? I know that you can drop tables and indexes sometimes kept.

**Pull request merged**